src/components/RouteWrapper.tsx

```diff
@@ -543,6 +544,13 @@ export const ROUTES: RouteProperties[] = [
             "The traceroute utility displays the route used by IP packets on their way to a specified network (or Internet) host.",
         category: "Network Scanning and Enumeration",
     },
+    {
+        name: "TShark",
+        path: "/tools/TShark",
+        element: <TShark />,
+        description: "Captures data packet's info and display in different fromats.",
```

Comment:
Recommend changing to: 
```suggestion
        description: "Captures data packets' information and displays in different formats.",
```
Assumes data packets should be plural, information rather than info (info is a bit casual) and there is a typo in formats. 

- [ ] Fix description